techlibs/ice40/abc_u.box

```diff
@@ -0,0 +1,113 @@
+# From https://github.com/cliffordwolf/icestorm/blob/be0bca0/icefuzz/timings_up5k.txt
+
+# NB: Inputs/Outputs must be ordered alphabetically
+
+# Inputs: C D
+# Outputs: Q
+SB_DFF 1 0 2 1
+- -
+
+# Inputs: C D E
+# Outputs: Q
+SB_DFFE 2 0 3 1
+- - -
+
+# Inputs: C D R
+# Outputs: Q
+SB_DFFSR 3 0 3 1
+- - -
+
+# Inputs: C D R
+# Outputs: Q
+SB_DFFR 4 0 3 1
+- - -
```

Comment:
The Xilinx equivalent, FDCE, has a path from async clear to out, whereas this doesn't - is there a reason for the difference?

Reply:
No intentional difference. I guess there wasn't an async clear that delay handy in https://github.com/cliffordwolf/icestorm/blob/be0bca0230d6fe1102e0a360b953fbb0d273a39f/icefuzz/timings_hx1k.txt

Also, not sure we/ABC can model async clears on flops...

Reply:
If you want the timing, it's the sr to lcout IOPATH.

I know abc9 has some internal modelling of set/resets, but I think that's for sync set/reset using the complex flop pattern - I can't see how you'd model an async one this way, using a whitebox and an AIG flipflop

Reply:
My understanding is that it can't be modelled using ABC9 either. I've removed this async timing from the Xilinx box.